apps/main/src/loan/components/PageLoanCreate/Page.tsx

```diff
@@ -151,7 +143,7 @@ const Page = () => {
 
   const TitleComp = () => (
     <AppPageFormTitleWrapper>
-      <Title>{displayName || getTokenName(llamma).collateral}</Title>
+      <Title>{market?.id === 'sfrxeth2' ? 'sfrxETH v2' : getTokenName(market).collateral}</Title>
```

Comment:
This logic came from the old collateral store, this was the only exception. Couldn't be bothered to make a helper function just for this.

Reply:
This case is also handled here more generically
https://github.com/curvefi/curve-frontend/blob/75c6ce5eab9f9c2d5cce023f952b740d046df25f/apps/main/src/llamalend/entities/llama-markets.ts#L256-L258

Reply:
I feel like we should add ` vN` for markets ending with a nr

Reply:
I agree, but the name generation is a bit too internal in llama-markets to re-use outside. I gave it a quick try but quickly ended up having refactor even more stuff, out of scope. Added a TODO for generalization later, not the point of this PR anyway.

Reply:
We don't even have this for lending markets and its only for the title when expanding charts 😭 
Super not important